recipes/install.rb

```diff
@@ -92,6 +92,7 @@
   docroot docroot
   es_server node['kibana']['es_server']
   kibana_port node['kibana']['java_webserver_port']
-  template 'kibana-nginx_file.conf.erb'
+  template node['kibana']['nginx']['template']
```

Comment:
The template resource you referred seems to create kibana config and not nginx/apache config as kibana_web resource does.